Add GDAL, versions 1.10.1 and 1.11.1_3

gmisra opened the pull request:
GDAL (http://www.gdal.org/) is a library package, and some of the projects that depend on it are version-sensitive, so being able to explicitly install a specific version should be helpful to others.

DomT4 commented:
Per lack of reply to Jack's comments, closing. Feel free to reopen a new PR if you're still interested.

gmisra commented:
Somehow I missed the original notifications, I am definitely still interested. I have created a new PR #675 that addresses the comments.

This is my first foray into homebrew-ing, thanks for your patience.
